packages/tasit-contracts/build/contracts/Migrations.json

```diff
@@ -0,0 +1,1498 @@
+{
+  "contractName": "Migrations",
+  "abi": [
```

Comment:
Do we want to parse this file to get the abi that we then pass into ethers? Or is there a cleaner way to achieve that?

Reply:
Maybe the parse will be necessary to interact with more complex contracts, but for now, [human-readable ABI](https://blog.ricmoo.com/human-readable-contract-abis-in-ethers-js-141902f4d917) is working fine.

Reply:
Good point. I haven't thought about this yet. We'll need to read the contract ABI, I'm not sure how the better way to do that 1) Reading the build file from `tasit-contracts` or 2) Having static ABI files saved inside the project that will use it.
The second option makes more sense to me, what do you think?

Reply:
I think static human-readable ABIs for the projects we support (and ERC standard static ABIs for those we don't explicitly support but implement that interface) is a sensible place to start. That way it will be sure to draw our attention to it when the ABI changes. If it becomes a regular problem to keep that updated we can look into automation at that point.

Reply:
Here's a related tool to check out for inspiration https://github.com/joincolony/trufflepig